bitwarden_license/src/Scim/Groups/PutGroupCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Scim.Context;
+using Bit.Scim.Groups.Interfaces;
+using Bit.Scim.Models;
+
+namespace Bit.Scim.Groups;
+
+public class PutGroupCommand : IPutGroupCommand
+{
+    private readonly IGroupRepository _groupRepository;
+    private readonly IGroupService _groupService;
+    private readonly IScimContext _scimContext;
+
+    public PutGroupCommand(
+        IGroupRepository groupRepository,
+        IGroupService groupService,
+        IScimContext scimContext)
+    {
+        _groupRepository = groupRepository;
+        _groupService = groupService;
+        _scimContext = scimContext;
+    }
+
+    public async Task<Group> PutGroupAsync(Guid organizationId, Guid id, ScimGroupRequestModel model)
+    {
+        var group = await _groupRepository.GetByIdAsync(id);
+        if (group == null || group.OrganizationId != organizationId)
+        {
+            throw new NotFoundException("Group not found.");
+        }
+
+        group.Name = model.DisplayName;
+        await _groupService.SaveAsync(group);
+        await UpdateGroupMembersAsync(group, model);
+
+        return group;
+    }
+
+    private async Task UpdateGroupMembersAsync(Group group, ScimGroupRequestModel model)
+    {
+        if (_scimContext.RequestScimProvider != Core.Enums.ScimProviderType.Okta)
+        {
+            return;
+        }
+
+        if (model.Members == null)
+        {
+            return;
+        }
+
+        var memberIds = new List<Guid>();
+        foreach (var id in model.Members.Select(i => i.Value))
+        {
+            if (Guid.TryParse(id, out var guidId))
+            {
+                memberIds.Add(guidId);
+            }
+        }
+
+        if (!memberIds.Any())
+        {
+            return;
+        }
```

Comment:
`skipIfEmpty` was set to `false` here, so this block should never evaluate, right? I think we need to delete this `if` block entirely, it doesn't apply to PUT operations.

Reply:
Well spotted! I've also added a check to the userIds passed to IGroupRepository on the unit test.